assign1/assign1-sol/index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html lang="en" ng-app='LunchCheck' ng-controller='LunchCheckController'>
+  <head>
+    <title>Lunch Checker</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="angular.min.js"></script>
+    <script src="app.js"></script>
+    <link rel="stylesheet" href="styles/bootstrap.min.css">
+    <style>
+      .message { font-size: 1.3em; font-weight: bold; }
+    </style>
+  </head>
+<body>
+   <div class="container">
+     <h1>Lunch Checker</h1>
+
+         <div class="form-group">
+             <input id="lunch-menu" type="text"
+             placeholder="list comma separated dishes you usually have for lunch"
+             class="form-control" ng-model="inputString" ng-class="lunch.inputType">
```

Comment:
And this `lunch.inputString`? It's always good practice to have a dot in your `ng-model`:
http://stackoverflow.com/questions/17606936/why-dont-the-angularjs-docs-use-a-dot-in-the-model-directive